examples/resources/bloxone_dhcp_server/resource.tf

```diff
@@ -26,3 +26,81 @@ resource "bloxone_dhcp_server" "example_with_options" {
     }
   ]
 }
+
+data "bloxone_dns_hosts" "example_dns_host" {
+  filters = {
+    name = "DNSHost01"
+  }
+}
+
+resource "bloxone_dns_view" "example_dns_view" {
+  name = "example_dns_view"
+
+  # Other Optional fields
+  comment = "An example view"
+  tags = {
+    site = "Site A"
+  }
+
+}
+
+resource "bloxone_dns_auth_zone" "example_auth_zone" {
+  fqdn         = "domain.com."
+  primary_type = "cloud"
+
+  # Other optional fields
+  comment = "Example of an Authoritative Zone"
+  tags = {
+    site = "Site A"
+  }
+  internal_secondaries = [
+    {
+      host = data.bloxone_dns_hosts.example_dns_host.results.0.id
+    },
+  ]
+  view = bloxone_dns_view.example_dns_view.id
+}
+
+resource "bloxone_dns_auth_zone" "example_reverse_zone" {
+  comment            = "This is a comment about the resource."
+  disabled           = false
+  fqdn               = "168.192.in-addr.arpa."
+  primary_type       = "cloud"
+  gss_tsig_enabled   = false
+  initial_soa_serial = 1
+  internal_secondaries = [
+    {
+      host = data.bloxone_dns_hosts.example_dns_host.results.0.id
+    },
+  ]
+  view = bloxone_dns_view.example_dns_view.id
+}
+
+resource "bloxone_dhcp_server" "example_with_ddns" {
```

Comment:
please add to the existing resource. `bloxone_dhcp_server.example`

Reply:
done